client/components/data/purchases/manage-purchase/index.jsx

```diff
@@ -35,7 +35,9 @@ const ManagePurchaseData = React.createClass( {
 		component: React.PropTypes.func.isRequired,
 		purchaseId: React.PropTypes.string.isRequired,
 		sites: React.PropTypes.object.isRequired,
-		destinationType: React.PropTypes.string
+		destinationType: React.PropTypes.string,
+		loadingPlaceholder: React.PropTypes.func,
+		isDataLoading: React.PropTypes.func
```

Comment:
We could also reference `isDataLoading` directly here. I guess both way works the same ;)